structuralalignment/superposition/mmligner.py

```diff
@@ -103,18 +104,17 @@ def _parse(self, output):
         for line in output.splitlines():
             if line.startswith(b"RMSD"):
                 rmsd = float(line.split()[2])
-            # coverage may still be relevant?
-            # elif line.startswith(b"Coverage"):
-            #   coverage = float(line.split()[2])
+            elif line.startswith(b"Coverage"):
+                coverage = float(line.split()[2])
 
             elif line.startswith(b"I(A & <S,T>)"):
                 ivalue = float(line.split()[4])
 
         alignment = fasta.FastaFile()
 
         return {
-            "superposed": None,
-            "scores": {"rmsd": rmsd, "score": ivalue},
+            "superposed": atomium.data.File("p_superposed__1.pdb").model,
```

Comment:
You can use `atomium.open(..path..).model`

Reply:
Oh okay, thanks for the fast reply!